ridgeback_navigation/launch/amcl_demo.launch

```diff
@@ -1,4 +1,5 @@
 <launch>
+  <arg name="scan_topic" default="front/scan" />
```

Comment:
Shouldn't this be `  <arg name="scan_topic" default="$(eval optenv('RIDGEBACK_LASER_TOPIC', 'front/scan'))" />`

Reply:
From the description of the PR: "Unlike the other platforms, we don't have a RIDGEBACK_LASER_TOPIC env var to leverage, so stick with front/scan as the default."

Reply:
Ops, thanks for the clarification.